dll/win32/shell32/lang/de-DE.rc

```diff
@@ -952,7 +952,7 @@ BEGIN
     IDS_EDIT_VERB "Bearbeiten"
     IDS_FIND_VERB "Finden"
     IDS_PRINT_VERB "Drucken"
-    IDS_CMD_VERB "Command Prompt hier"
+    IDS_CMD_VERB "Eingabeaufforderung hier"
```

Comment:
I would really love if we wouldn't translate that further than it currently is. Reasons:
1.) C... is easily associated in the brain with cmd, while E... is not.
2.) The context menu might get wider than necessary with the long german word Eingabeaufforderung.
3.) The widespread Powertoys tool also calls this "Command prompt here" without localization.

If you really have to change that out of some intrinsic motivation, then please at least use
"CMD hier" or "Cmd hier".
But please not "Eingabeaufforderung hier". Thank you!

Reply:
> Eingabeaufforderung 

Indeed, the noun is not long enough for a good translation :rofl:

Reply:
Just to note, if we want to be consistent, these cases would need to be fixed: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=Eingabeaufforderung

Reply:
interesting idea. I would have expected more matches actually. That would indeed also solve uglyness like that:
![image](https://user-images.githubusercontent.com/33393466/197608923-074735fb-2184-4ed5-99c4-f2906ab63143.png)

Reply:
I changed it to Cmd.
I second your opinion that "Eingabeaufforderung" is too long. ("lovely" German language).
But "Command Prompt hier" sounds Denglisch.
The opinion that most ppl are looking for "cmd" do I share only partially. For me and you it's cmd, but for average Joe it might not. (In fact I never do click somewhere when opening a prompt, I press win+R and type cmd)

So I think it's a good compromise.